packages/perseus/src/__tests__/renderer-api_test.jsx

```diff
@@ -190,13 +166,13 @@ describe("Perseus API", function () {
                     // $FlowFixMe[prop-missing]
                     ReactDOM.findDOMNode(renderer).querySelector("input");
 
-                expect($(input).hasClass(ClassNames.FOCUSED)).toBe(false);
+                expect($(input).attr("class").includes("focused")).toBe(false);
 
                 TestUtils.Simulate.focus(input);
-                expect($(input).hasClass(ClassNames.FOCUSED)).toBe(true);
+                expect($(input).attr("class").includes("focused")).toBe(true);
 
                 TestUtils.Simulate.blur(input);
-                expect($(input).hasClass(ClassNames.FOCUSED)).toBe(false);
+                expect($(input).attr("class").includes("focused")).toBe(false);
```

Comment:
style nit: `jest-dom`'s `toHaveClass()` matcher could be used here instead of relying on jQuery.

Reply:
The class is a big string blob, I think I need this to do string matching

Reply:
Is that because the class is `<something>-focused-<some-other-thing>` as opposed to `focused` being surrounded by spaces?

Reply:
I wonder if [toHaveFocus()](https://github.com/testing-library/jest-dom#tohavefocus) would work. 🤔

Reply:
toHaveFocus doesn't work. And yes it's because of the big (GIANT) blob of class stuff that gets mashed together